guides/source/getting_started.md

```diff
@@ -388,9 +388,9 @@ create and read. The form for doing this will look like this:
 ![The new article form](images/getting_started/new_article.png)
 
 It will look a little basic for now, but that's ok. We'll look at improving the
-styling for it afterwards.
+styling for it afterward.
```

Comment:
Afterward is used in American English. Afterwards is used in English from outside the US.
I noticed that Rails documentation prefers American English:
http://edgeguides.rubyonrails.org/api_documentation_guidelines.html#english